docs/api/ref-api-and-product-schema-change-log.md

```diff
@@ -0,0 +1,66 @@
+# Reference: API and product schema change log
+
+This reference lists changes to the API and/or the product schema.
+
+## Introduction
+
+The schema of product data in the Open Food Facts database changes over time:
+- We often add new raw data fields (e.g. packaging data), or new computed fields (e.g. health and environmental scores).
+- Occasionally, we refactor some fields to improve how we structure some data (e.g. in the beginning we had a flat list of ingredients, then we added a nested ingredients structure to better represent sub-ingredients).
+
+We consider new fields to be non-breaking changes. When reading product data, make sure that your implementation will ignore new fields and not break.
+
+We consider changed fields to be breaking changes. To maintain backward compatibility, we increase the API version number when there is a breaking change, and we try to serve the old structure when a lower API version is requested by a client.
+
```

Comment:
Maybe add something about the fact that the API might return undocumented fields, you should not rely on them if they are not part of the documentation (but ask on #api channel or opening an issue).

Reply:
Good idea, doing that.